modules/Bio/EnsEMBL/Analysis/Hive/Config/Genome_annotation_conf.pm

```diff
@@ -1910,180 +1909,11 @@ sub pipeline_analyses {
-      -parameters => {
-        beekeeper_script => $self->o('hive_beekeeper_script'),
-      },
-      -rc_name      => 'default',
-      -max_retry_count => 1,
-    },
-
-
-    {
-      -logic_name => 'skip_rnaseq_db',
-      -module => 'Bio::EnsEMBL::Hive::RunnableDB::SystemCmd',
-      -parameters => {
-        cmd => 'if [ #skip_rnaseq# -eq 0 ]; then exit 0; else exit 42;fi',
-        return_codes_2_branches => {'42' => 2},
-      },
-      -rc_name => 'default',
       -flow_into  => {
-        1 => ['create_rnaseq_db_pipeline_job'],
-      },
-    },
-
-    {
-      -logic_name => 'create_rnaseq_db_pipeline_job',
-      -module => 'Bio::EnsEMBL::Hive::RunnableDB::JobFactory',
-      -parameters => {
-        column_names => ['meta_pipeline_db', 'ehive_url', 'pipeline_name', 'external_link'],
-        inputlist => [[$rnaseq_db_pipe_db, $rnaseq_db_pipe_url, 'rnaseq_db_'.$self->o('production_name'), $rnaseq_db_guihive]],
-        guihive_host => $self->o('guihive_host'),
-        guihive_port => $self->o('guihive_port'),
-      },
-      -rc_name => 'default',
-      -max_retry_count => 0,
-      -flow_into => {
-        2 => ['initialise_rnaseq_db'],
-      }
-    },
-
-    {
-      -logic_name => 'initialise_rnaseq_db',
-      -module     => 'Bio::EnsEMBL::Analysis::Hive::RunnableDB::HiveMetaPipelineInit',
-      -parameters => {
-        hive_config => $self->o('hive_rnaseq_db_config'),
-        databases => ['rnaseq_db', 'rnaseq_refine_db', 'rnaseq_blast_db', 'dna_db'],
-        rnaseq_db => $self->o('rnaseq_db'),
-        rnaseq_refine_db => $self->o('rnaseq_refine_db'),
-        rnaseq_blast_db => $self->o('rnaseq_blast_db'),
-        dna_db => $self->o('dna_db'),
-        enscode_root_dir => $self->o('enscode_root_dir'),
-        extra_parameters => {
-          output_path => $self->o('output_path'),
-          user_r => $self->o('user_r'),
-          dna_db_host => $self->o('dna_db_host'),
-          dna_db_port => $self->o('dna_db_port'),
-          databases_host => $self->o('databases_host'),
-          databases_port => $self->o('databases_port'),
-          release_number => $self->o('release_number'),
-          production_name => $self->o('production_name'),
-          dbname_accession => $self->o('dbname_accession'),
-          species_name => $self->o('species_name'),
-          registry_host => $self->o('registry_host'),
-          registry_port => $self->o('registry_port'),
-          registry_db => $self->o('registry_db'),
-          assembly_name => $self->o('assembly_name'),
-          assembly_accession => $self->o('assembly_accession'),
-          annotation_source => $self->o('annotation_source'),
-          sanity_set => $self->o('sanity_set'),
-        },
-      },
-      -rc_name      => 'default',
-      -max_retry_count => 0,
-      -flow_into => {
-        1 => ['run_rnaseq_db'],
+	              1 => ['update_assembly_registry_status'],
```

Comment:
to check indentation you can run perltidy -b -i=2 -l=0 -ci=2 -it=2 -isbc -nbbc --freeze-newlines filename

Reply:
ok it is a bit difficult to check from here, I just saw 1913 with too many tabs
Thanks